lib/Server.js

```diff
@@ -694,11 +694,15 @@ class Server {
         }
       });
 
+      if (this.clientLogLevel) {
+        this.sockWrite([connection], 'log-level', this.clientLogLevel);
+      }
+
       if (this.hot) {
         this.sockWrite([connection], 'hot');
       }
 
-      if (this.options.liveReload !== false) {
+      if (this.options.liveReload) {
```

Comment:
Can be breaking change for non boolean value, let's avoid this in this PR

Reply:
ok, I'll change it at next major version.

Reply:
Yes